### PR TITLE
Add new partner SquareShift

### PR DESCRIPTION
**Pull request by elangoshifts**
### Description
Adds a new partner SquareShift
 
### Check List
- [x] Commits are signed per the DCO using --signoff

Screenshot 2022-05-25 at 8 13 47 PM

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _partners/squareshift.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+name: SquareShift
+# upload your logo to the following directory - must be square
+logo: 'https://static.wixstatic.com/media/4d2638_9099aa345f0848a18f4d8ebf24ae0fb5~mv2.png'
```
Comment (krisfreedain) [inline _partners/squareshift.md:4]: Please submit the logo directly as part of the pull request - example:

logo: '/assets/media/partners/squareshift.png'

**Review by krisfreedain — CHANGES_REQUESTED** [review]
@elangoshifts - small change needed. thanks

File: _partners/squareshift.md (hunk 1 above)
Comment (elangoshifts) [reply _partners/squareshift.md]: I initially mis-understood that relative url has to be on our domain, then testing locally help identify that it has to be relative to Opensearch site. This has been addressed now. Thank you

Comment (krisfreedain) [reply _partners/squareshift.md]: Looks good!

**Pull request merged**